store/plugin-commands-store-search/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@pnpm/plugin-commands-store-search",
+  "version": "0.0.1",
+  "description": "The visualization store commands of pnpm",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "files": [
+    "lib",
+    "!*.map"
+  ],
+  "engines": {
+    "node": ">=16.14"
+  },
+  "scripts": {
+    "lint": "eslint \"src/**/*.ts\" \"test/**/*.ts\"",
+    "_test": "jest",
+    "test": "pnpm run compile && pnpm run _test",
+    "prepublishOnly": "pnpm run compile",
+    "compile": "tsc --build && pnpm run lint --fix"
+  },
+  "repository": "https://github.com/pnpm/pnpm/blob/main/store/plugin-commands-store-search",
+  "keywords": [
+    "pnpm8",
+    "pnpm"
+  ],
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/pnpm/pnpm/issues"
+  },
+  "homepage": "https://github.com/pnpm/pnpm/blob/main/store/plugin-commands-store-search#readme",
+  "devDependencies": {
+    "@pnpm/config": "workspace:*",
+    "@pnpm/parse-wanted-dependency": "workspace:*",
+    "@pnpm/pick-registry-for-package": "workspace:*",
+    "@pnpm/client": "workspace:*",
+    "@pnpm/store.cafs": "workspace:*",
+    "@pnpm/lockfile-types": "workspace:*",
+    "@pnpm/store-path": "workspace:*",
+    "@pnpm/graceful-fs": "workspace:*",
+    "@pnpm/plugin-commands-store-search": "workspace:*",
+    "@pnpm/prepare": "workspace:*",
+    "load-json-file": "^6.2.0",
+    "tempy": "^1.0.1",
+    "execa": "npm:safe-execa@0.1.2"
+  },
+  "dependencies": {
+    "@pnpm/error": "workspace:*",
+    "@pnpm/types": "workspace:*",
+    "render-help": "^1.0.3",
+    "chalk": "^4.1.2"
+  },
```

Comment:
These dependencies are broken. Some of the dev dependencies are used in runtime code, they should be moved to prod dependencies.